src/pages/SignUp.js

```diff
@@ -50,15 +51,30 @@ function SignUp() {
     return errors;
   };
 
+  const [selectedValue, setSelectedValue] = React.useState('researcher');
+  
+  const handleChange = (event) => {
+      setSelectedValue(event.target.value);
+    }
```

Comment:
(optional) nit: indentation seems a bit off. also, can we remove the curly braces so just
`const handleChange = (event) => setSelectedValue(event.target.value);`

also, are we able to run linter so it will fix the format automatically?